### DISCUSSION TITLE
Manual/script-CanvasScaler

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#686] [Notes]
# Manual/script-CanvasScaler

https://docs.unity3d.com/Manual/script-CanvasScaler.html

**Comment by jowsey** [comment]
If you're trying to multiply UI pixel co-ordinates based on resolution scale, you're looking for `Canvas#scaleFactor`, not `CanvasScaler#scaleFactor`. Reading the `localScale` of the Canvas should also get you the same number.
